src/main/java/com/github/manolo8/darkbot/core/objects/Gui.java

```diff
@@ -84,7 +84,8 @@ public void hover(int plusX, int plusY) {
 
     public boolean show(boolean value) {
         if (trySetShowing(value)) {
-            if (minimized.address != 0) API.mouseClick((int) minimized.x + 5, (int) minimized.y + 5);
+            if (value && minimized.address != 0) API.mouseClick((int) minimized.x + 5, (int) minimized.y + 5);
+            else if (!value) click(5, 5);
```

Comment:
We can't make this change or overlapped windows will not be able to close, i'm pretty sure windows overlapping eachother is more common than windows overlapping the minimized bar